aws/resource_aws_lb_target_group.go

```diff
@@ -294,6 +308,10 @@ func resourceAwsLbTargetGroupCreate(d *schema.ResourceData, meta interface{}) er
 		}
 		params.Port = aws.Int64(int64(d.Get("port").(int)))
 		params.Protocol = aws.String(d.Get("protocol").(string))
+		switch d.Get("protocol").(string) {
+		case elbv2.ProtocolEnumHttp, elbv2.ProtocolEnumHttps:
```

Comment:
I personally like `switch` even in simple logic since it's so clean. Some suggest `if` is better here but I like it.

Reply:
Same here. I forget sometimes when I'm deep in what I'm doing though.